app/controllers/welcome_controller.rb

```diff
@@ -1,11 +1,11 @@
 class WelcomeController < ApplicationController
 
-	def home
-		@movies_podium = [ Movie.rank_by_attendance(1), Movie.rank_by_attendance(2), Movie.rank_by_attendance(3)]
-		@oldest_movie = Movie.oldest
-		@latest_movie = Movie.latest
+  def home
+    @movies_podium = [ Movie.rank_by_attendance(1), Movie.rank_by_attendance(2), Movie.rank_by_attendance(3)]
```

Comment:
Same as for director, this podium could be move (and test) easily in a method of an other object